lib/ethpy/ethpy/hyperdrive/deploy.py

```diff
@@ -229,79 +226,62 @@ def _deploy_hyperdrive_factory(
     (base_token_contract, factory_token_contract, pool_contract_address): tuple[Contract, Contract, ChecksumAddress]
         Containing the deployed base token, factory, and the pool contracts/addresses.
     """
-    # args = [name, symbol, decimals, admin_addr, isCompetitionMode]
-    args = ["Base", "BASE", 18, ADDRESS_ZERO, False]
-    base_token_contract_addr, base_token_contract = deploy_contract(
-        web3,
-        abi=abis["ERC20Mintable"],
-        bytecode=bytecodes["ERC20Mintable"],
-        deploy_account_addr=deploy_account_addr,
-        args=args,
+    erc20args = ERC20MintableContract.ConstructorArgs("Base", "BASE", 18, ADDRESS_ZERO, False)
+    base_token_contract = ERC20MintableContract.deploy(w3=web3, account=deploy_account_addr, constructorArgs=erc20args)
+
+    pool = MockERC4626Contract.deploy(
+        w3=web3,
+        account=deploy_account_addr,
+        constructorArgs=MockERC4626Contract.ConstructorArgs(
```

Comment:
And this would be:

```python
constructorArgs=MockERC4626Contract.ConstructorArgs(
    base_token_contract.address, # asset
    "Delvnet Yield Source", # name
    "DELV", # symbol
    initial_variable_rate.scaled_value, # initialRate
    ADDRESS_ZERO, # admin 
    False, # isCompetitionMode
),
 ```

Reply:
using kwarg style instead.